src/libs/Environment/Environment.js

```diff
@@ -31,8 +37,21 @@ function getEnvironmentURL() {
     });
 }
 
+/**
+ * Get the corresponding oldDot URL based on the environment we are in
+ *
+ * @returns {Promise}
+ */
+function getOldDotEnvironmentURL() {
```

Comment:
Do we know if this should depend on the staging api toggle (see Preferences / Settings / Test Settings)

Should this return the PROD Old Dot URL, if the API toggle is set to production?

<img width="363" alt="Screenshot 2023-02-17 at 16 12 21" src="https://user-images.githubusercontent.com/12156624/219678442-788ac3d5-9cea-4f42-b8aa-1a5a611fe119.png">

Reply:
Yes!
> if the API toggle is set to production?

I guess this is when "Use staging server" is toggled off

Reply:
Wait, looking into the above problem, we should clarify this.

Reply:
> > if the API toggle is set to production?
> 
> I guess this is when "Use staging server" is toggled off

Correct

---

> Do we know if this should depend on the staging api toggle (see Preferences / Settings / Test Settings)
>>  Wait, looking into the above [problem](https://github.com/Expensify/App/pull/14944#discussion_r1109856517), we should clarify this.

I'm not sure either, can you ask on the issue related to the current PR, or open a slack thread about it